### DISCUSSION TITLE
✅ Typewriter (Rotating) Effect for Heading

### DISCUSSION THREAD

**Discussion by jay639** [elementor/elementor#22798] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

We need to be depend on the third party plugins for this feature. This is must feature that new elementor should have. 

With this feature people can create an engaging content and stunning design.

### Describe the solution you'd like

Typewriter (Text Rotation) is available in many popular themes and page builders including Avada.

### Describe alternatives you've considered

Elementor adds on plugin like ultimate elements 

### Additional context

Please find the reference link.
https://bit.ly/3PdJKfb

**Comment by jay639** [comment]
It's available in Elementor pro as Animated Headline

**Comment by nicholaszein** [comment]
Hi @jay639.

This feature is already available in Elementor Pro as you pointed out. Please refer to the link below for more info.

https://elementor.com/help/animated-headline-pro/
